### PR TITLE
Azure: Add TDX deployment instructions

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

**Comment by surajssd** [comment]
The changes are deployed here: https://deploy-preview-41--frolicking-manatee-96c0c8.netlify.app/docs/cloud-api-adaptor/azure/

File: content/en/docs/cloud-api-adaptor/azure/_index.md [hunk 1]
```diff
@@ -31,13 +31,47 @@ There are a bunch of steps that require you to be logged into your Azure account
...
+
+{{% /tab %}}
+
+{{% tab header="Intel TDX" %}}
+
+```bash
+export AZURE_REGION="eastus2"
+```
+
+> **Note:** We selected the `eastus` region as it not only offers Intel TDX machines but also has prebuilt pod VM images readily available.
```
[the remark this answers is not in the document]
Comment (surajssd) [reply content/en/docs/cloud-api-adaptor/azure/_index.md:60]: Again, copy-pasta, good catch, Magnus 😆

[the remark this answers is not in the document]
Comment (surajssd) [reply content/en/docs/cloud-api-adaptor/azure/_index.md:60]: On line # 57 I say eastus2 while the note says `eastus` it is erroneously not-updated!

**Review by mkulke — APPROVED** [review]

**Review by kartikjoshi21 — APPROVED** [review]
Tabs looks cool !! Thanks @surajssd

**Pull request merged**